manifests/g/Guru3D/Afterburner/4.6.5.Beta4/Guru3D.Afterburner.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Guru3D.Afterburner
+PackageVersion: 4.6.5.Beta4
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: nullsoft
+  NestedInstallerFiles:
+  - RelativeFilePath: MSIAfterburnerSetup465Beta4.exe
+  InstallerUrl: https://download-eu2.guru3d.com/afterburner/[Guru3D.com]-MSIAfterburnerSetup465Beta4Build16358.zip
+  InstallerSha256: 87E15ED6C587B7A187D29A73452FC7F581E72FB1634199AA82D8C64F9F6FDF01
+  InstallModes:
+  - silent
+  - interactive
+  InstallerSwitches:
+    Silent: /S
+    InstallLocation: /D
+  UpgradeBehavior: install
```

```suggestion
  UpgradeBehavior: install
  AppsAndFeaturesEntries: 
  - DisplayName: MSI Afterburner 4.6.5 Beta 4
    DisplayVersion: 4.6.5 Beta 4
```
AppsAndFeaturesEntries are needed for proper matching due to the spaces